alpha-core/src/test/java/at/ac/tuwien/kr/alpha/core/solver/AggregatesTest.java

```diff
@@ -335,4 +335,36 @@ public void aggregateSumGlobalVariable(RegressionTestConfig cfg) {
 				"full(2)");
 	}
 
+	@AggregateRegressionTest
+	public void aggregateRightHandTermCountGreater(RegressionTestConfig cfg) {
+		String program = "p(1)." + 
```

Comment:
@AntoniusW regarding #301: Here, IMO [JEP-378: Text Blocks](https://openjdk.java.net/jeps/378) would make this look prettier.